workflows/proteomics/clinicalmp/clinicalmp-verification/iwc_clinicalmp_verification_workflow.ga

```diff
@@ -0,0 +1,933 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "WF3- Peptide verification/validaiton workflow",
+    "comments": [],
+    "creator": [
+        {
+            "class": "Organization",
+            "identifier": "0000-0001-9818-0537",
+            "name": "Subina Mehta"
+        }
+    ],
+    "format-version": "0.1",
+    "license": "CC-BY-4.0",
+    "name": "iwc_clinicalmp_verification_workflow",
```

Comment:
```suggestion
    "name": "iwc_clinicalmp_verification_workflow",
```
This is the name users see in the workflow list and in https://iwc.galaxyproject.org/. Could you use something more targeted at users, like `Clinical Metaproteomics Verification` ... something that gives users an idea what this workflow is about ? Ideally this would include what is being verified ?

Reply:
Removed underscores and changed the name.